src/core_atmosphere/Registry.xml

```diff
@@ -1370,6 +1374,12 @@
                 <var name="defc_b" type="real" dimensions="maxEdges nCells" units="unitless"
                      description="Coefficients for computing the diagonal components of the horizontal deformation"/>
 
+                <var name="cell_gradient_coef_x" type="real" dimensions="maxEdges nCells" units="m^-1"
+                     description="Coefficients for computing the x derivative of a cell-centered variable"/>
+
+                <var name="cell_gradient_coef_y" type="real" dimensions="maxEdges nCells" units="m^-1"
+                     description="Coefficients for computing the y derivative of a cell-centered variable"/>
```

Comment:
Are these actually zonal and meridional gradients? If not, how are the x- and y-directions defined?

Reply:
Actually, I just read the commit message for ddb57a8f, and it seems that these are for zonal and meridional derivatives. Should we change the descriptions (or even the names) to make this clear?